crates/data_components/src/s3_vectors/vector_table.rs

```diff
@@ -439,8 +439,9 @@ impl S3VectorsTable {
         }
 
         tracing::info!(
-            "S3 Vectors Index updated; records={} records, duration={duration:?}",
-            vectors.len(),
+            "S3 Vectors Index {:?} updated; records={records}, duration={duration:?}",
```

Comment:
```suggestion
            "S3 Vectors Index {} updated; records={records}, duration={duration:?}",
```

info traces should not include debug output

Reply:
Yes, and it may be `None` even when there is a valid index.

Reply:
We should `impl Display for S3VectorIdentifier` and then use that